### ISSUE TITLE
error generating in-repo-engine

### ISSUE THREAD

**Issue by marcemira** [dgeb/ember-engines#217]
Using ember-cli 2.9.0-beta.2, trying to generate an in-repo-engine I get the following with current ember-engines v0.3.2:
```
$ ember g in-repo-engine my-engine
installing in-repo-engine
installing lib
  identical lib/.jshintrc
  identical lib/my-engine/addon/engine.js
  identical lib/my-engine/addon/routes.js
  identical lib/my-engine/addon/templates/application.hbs
  identical lib/my-engine/config/environment.js
  identical lib/my-engine/index.js
  identical lib/my-engine/package.json
this._generatePackageJson is not a function
TypeError: this._generatePackageJson is not a function
    at CoreObject.afterInstall (/path/to/my/addon/node_modules/ember-cli/blueprints/in-repo-addon/index.js:21:10)
    at CoreObject.afterInstall (/path/to/my/addon/node_modules/ember-engines/blueprints/in-repo-engine/index.js:68:30)
    at tryCatch (/path/to/my/addon/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:538:12)
    at invokeCallback (/path/to/my/addon/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:553:13)
    at publish (/path/to/my/addon/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:521:7)
    at flush (/path/to/my/addon/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:2370:5)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```
```
$ ember --version
ember-cli: 2.9.0-beta.2
node: 6.5.0
os: linux x64
```
Tried using `ember g in-repo-addon my-addon` and it works, but I would really love to use this blueprint instead. Thanks!

**Comment by nathanhammond** [comment]
Yep, this changed here:
https://github.com/ember-cli/ember-cli/commit/79a4b40a1e7c0c9cb356fc2bcf3f0348a0191eb3#diff-006bfafbcbf2e60bf76521f12ad0cde8L19

This file needs to be changed:
https://github.com/dgeb/ember-engines/blob/master/blueprints/in-repo-engine/index.js

@marcemira Would you be willing to modify the blueprint in ember-engines to match the upstream changes to `in-repo-addon`?

**Comment by marcemira** [comment]
@nathanhammond I will :muscle:, thanks!

**Comment by knownasilya** [comment]
This should be closed @nathanhammond

**Comment by trentmwillis** [comment]
Looks to have been fixed via #219
